### PR TITLE
OSD-14200 Add Max Concurrent Reconcile limit

### PR DESCRIPTION
**Pull request by reedcort**
# What is being added?
Adds Max Concurrent Reconcile limit field in the configmap for the accountpool validation controller

## Checklist before requesting review
- [x] I have tested this locally
- [ ] I have included unit tests
- [ ] I have updated any corresponding documentation

Ref [OSD-14200](https://issues.redhat.com//browse/OSD-14200)

### CODE REVIEW

File: controllers/validation/accountpool_validation_controller.go
Comment (AlexVulaj) [inline controllers/validation/accountpool_validation_controller.go:1]: Can we revert the changes in this file? I think it's better to have this string defined in one consistent place instead of repeating it in multiple places.

Comment (rafael-azevedo) [reply controllers/validation/accountpool_validation_controller.go:1]: +1

Comment (reedcort) [reply controllers/validation/accountpool_validation_controller.go:1]: Added a the var back but changed the name, because the name 'controllerName' is already being used by the 'accountValidation' controller
